Library/Homebrew/cmd/reinstall.rb

```diff
@@ -59,7 +63,9 @@ def reinstall
         next
       end
       Migrator.migrate_if_needed(f)
-      reinstall_formula(f)
+      reinstall_formula(f) do |fi|
```

Comment:
```suggestion
      reinstall_formula(f, interactive: args.interactive?)
```

Reply:
Nice suggestion!

I wasn't sure what to do here — deep inside `reinstall_formula` uses `ARGV.interactive?` so perhaps just adding the flag to the listed flags in `cmd/reinstall` is enough, and remove the `yield`? I notice `reinstall_formula` is only otherwise used in `cmd/upgrade` which also does not list `--interactive`.

Reply:
Yeah, just listing the switch in the switches is enough to make it work. I'll back out the rest.

Reply:
Sounds good!